tools/expotools/src/versioning/ios/transforms/postTransforms.ts

```diff
@@ -37,6 +37,10 @@ export function postTransforms(versionName: string): TransformPipeline {
         replace: new RegExp(`FB${versionName}ReactNativeSpec`, 'g'),
         with: 'FBReactNativeSpec',
       },
+      {
+        replace: new RegExp('([^\\w\\.])(Native\\w+Spec)([^\\w\\.])', 'g'),
```

Comment:
I guess `\\b` would work instead of `[^\\w\\.]` but I'm fine with leaving it as is 😉

Reply:
👍 I changed to `\\b` as it fixes a possible bug where the `Native…Spec` would start at the beginning of a line and we wouldn't find it.